index.js

```diff
@@ -1,21 +1,42 @@
+/*jshint node: true */
+'use strict';
+
 var path = require('path');
+var mergeTrees = require('broccoli-merge-trees');
+var Funnel = require('broccoli-funnel');
 
 module.exports = {
   name: 'ember-data-factory-guy',
 
+  treeForVendor: function() {
```

Comment:
Yes, this is including the `urijs` addon and including it in the build. I took inspiration from other addons https://github.com/bustlelabs/ember-mobiledoc-editor/blob/master/index.js and https://github.com/ember-animation/liquid-fire/blob/master/index.js.

Are you sure the factories are not already included in the build? If I run `ember s` and go to `localhost:4200/tests` then I can use all my factories and functions exported by `ember-data-factory-guy`.